VKappProject/VKappProject/Controllers/AvailableGroupsTableViewController.swift

```diff
@@ -34,33 +25,29 @@ final class AvailableGroupsTableViewController: UITableViewController {
 
     private var isSearching = false
 
-    // MARK: - Public Methods
-
-    func addGroup(_ group: [Group], complition: @escaping ((Group) -> Void)) {
-        availableGroups = availableGroups.filter { groups in
-            !group.contains { myGroup in
-                myGroup == groups
-            }
-        }
-        closureForGroup = complition
-    }
-
     // MARK: - Private Methods
 
-    private func chooseGroup(_ indexPath: IndexPath) {
-        let group = availableGroups[indexPath.row]
-        closureForGroup?(group)
-        navigationController?.popViewController(animated: true)
+    private func fetchSearchedGroupsRequest(searchedText: String) {
+        AF
```

Comment:
```suggestion
        AF
```
Это все надо делать в сетовом классе что б контролер ничего не знал о запросах